src/content-script/in-dapp-notification/index.ts

```diff
@@ -0,0 +1,105 @@
+import * as styles from './styles.module.css';
+
+function preloadImage(url: string): Promise<HTMLImageElement> {
+  return new Promise((resolve, reject) => {
+    const img = new Image();
+    img.src = url;
+    img.onload = () => resolve(img);
+    img.onerror = reject;
+  });
+}
+
+function clearNotifications() {
+  document
+    .querySelectorAll(`.${styles.notification}`)
+    .forEach((notification) => notification.remove());
+}
+
+function removeNotification(notification: HTMLElement) {
+  notification.classList.add(styles.fadeOut);
+  setTimeout(() => {
+    notification.remove();
+  }, 500);
```

Comment:
May be it can be easier to extract all these numbers as consts and use define them in one place to avoid miscalculation

Reply:
We decided not to remove the injected CSS, so using "in-place" constants for timeouts like this shouldn't be an issue